features/step_definitions/collection_steps.rb

```diff
@@ -323,3 +332,13 @@
   visit user_path(work.users.first)
   expect(page).not_to have_content(title)
 end
+
+Then "{string} has the {string} role in the collection {string}" do |byline, role, collection|
```

Comment:
This reads a bit more like a setup (`Given`) step. Can you rephrase it to "{string} should have the {string} role..." so it's clearer we're checking expected results, not providing background?